<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Params_WindowClass</class>
 <widget class="QDialog" name="Params_WindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Params_Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="formWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>175</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>175</height>
      </size>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="Name">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(220, 220, 220);
border: 2px solid;
border-color: rgb(10, 10, 10);</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="Name_Edit">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid;
border-color: rgb(10, 10, 10);
font: 900 10pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="Firm">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(220, 220, 220);
border: 2px solid;
border-color: rgb(10, 10, 10);</string>
        </property>
        <property name="text">
         <string>Firm</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="Firm_Edit">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid;
border-color: rgb(10, 10, 10);
font: 900 10pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="Seria">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(220, 220, 220);
border: 2px solid;
border-color: rgb(10, 10, 10);
</string>
        </property>
        <property name="text">
         <string>Seria</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="Seria_Edit">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid;
border-color: rgb(10, 10, 10);
font: 900 10pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="Height">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid;
border-color: rgb(10, 10, 10);
font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(220, 220, 220);</string>
        </property>
        <property name="text">
         <string>Height</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="Height_Edit">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid;
border-color: rgb(10, 10, 10);
font: 900 10pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="Width">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid;
border-color: rgb(10, 10, 10);
font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(220, 220, 220);</string>
        </property>
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="Width_Edit">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid;
border-color: rgb(10, 10, 10);
font: 900 10pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="Length">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid;
border-color: rgb(10, 10, 10);
font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(220, 220, 220);</string>
        </property>
        <property name="text">
         <string>Length</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="Length_Edit">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid;
border-color: rgb(10, 10, 10);
font: 900 10pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="Volume">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid;
border-color: rgb(10, 10, 10);
font: 900 12pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(220, 220, 220);</string>
        </property>
        <property name="text">
         <string>Weight</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="Weight_Edit">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid;
border-color: rgb(10, 10, 10);
font: 900 10pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);
background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Next_Button">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 2px solid;
border-color: rgb(10, 10, 10);
font: 900 10pt &quot;Segoe UI Black&quot;;
color: rgb(10, 10, 10);

background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Next &gt;&gt;&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
